# Topics and services names
find_objects_service: find_objects
input_cloud_topic: /camera/rgb/points
extract_dominant_plane_service: extract_dominant_plane

# Topics for optional output
found_clusters_topic: found_clusters
bounding_boxes_topic: bounding_boxes

# How many clouds to accumulate
number_of_merges: 25

# Spend at most this many seconds accumulating clouds
accumulation_timeout: 40

# Filtering parameters
filter_min_points: 140
filter_min_height: 0.01
